modules/ossm-upgrade-23-24-changes.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/upgrading-ossm.adoc
+
+:_content-type: CONCEPT
+[id="ossm-upgrade-23-24-changes_{context}"]
+= Upgrade changes from version 2.3 to version 2.4
+
+Upgrading the {SMProductShortName} control plane from version 2.3 to 2.4 introduces the following behavioral changes:
+
+* Support for Istio OpenShift Routing (IOR) has been deprecated. IOR functionality is still presently enabled, but it will be removed in a future release.
```

Comment:
```suggestion
* Support for Istio OpenShift Routing (IOR) has been deprecated. IOR functionality is still enabled, but it will be removed in a future release.
```

Reply:
Done